project/Dependencies.scala

```diff
@@ -93,7 +93,7 @@ object Dependencies {
   val workbenchLibsHash = "20f9225"
 
   val workbenchModelV  = s"0.15-${workbenchLibsHash}"
-  val workbenchGoogleV = s"0.21-${workbenchLibsHash}"
+  val workbenchGoogleV = s"0.21-805f2011-SNAP"
```

Comment:
I will migrate to a non-snap once the relevant workbench-libs PR is merged. There was a small bit of work to converge the Rawls and workbench-libs pub sub DAO implementations, as the Rawls one had progressed a bit over the years whereas workbench-libs stayed the same